### PR TITLE
Upgrading to latest requirements is eager

### PR DESCRIPTION
**Pull request by potiuk**
---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
The CRON job failing tonight had shown that we should have eager strategy in upgrade script - this way we will keep up with the latest requirements as seen by anyone trying to install airflow from the scratch. And our tests will be using it.

https://travis-ci.org/github/apache/airflow/builds/668343196

**Review by mik-laj — APPROVED** [review]

File: requirements/requirements-python3.6.txt [hunk 1]
```diff
@@ -176,7 +177,7 @@ lockfile==0.12.2
 Mako==1.1.2
 Markdown==2.6.11
 MarkupSafe==1.1.1
-marshmallow==3.5.1
+marshmallow==2.21.0
```
Comment (Fokko) [inline requirements/requirements-python3.6.txt:180]: This one is interesting, it is being downgraded?

**Review by Fokko — APPROVED** [review]
Love it

File: requirements/requirements-python3.6.txt (hunk 1 above)
Comment (potiuk) [reply requirements/requirements-python3.6.txt:180]: I think it could be because another requirement added constraint.

**Pull request merged**